proposals/nnnn-structured-concurrency.md

```diff
@@ -243,7 +243,7 @@ We can then re-implement `makeDinner` with only a task group:
 
 ```swift
 func makeDinnerTaskGroup() async throws -> Meal {
-  withTaskGroup(resultType: DinnerChildTask.self) { group in    
+  return try await Task.withGroup(resultType: DinnerChildTask.self) { group in    
```

Comment:
`resultType` should have `DinnerChild.self`

Reply:
Nice catch, will fix